### PR TITLE
Added missing factory upgrades and modified upgrade times

### PR DESCRIPTION
**Pull request by Goldmattress**
Added missing factory upgrades and modified upgrade times, loads of other things got tweaked as well.

### CODE REVIEW

File: configCivcraft.yml [hunk 1]
```diff
@@ -695,9 +697,9 @@ recipes:
   Craft_Anvils_Advanced:
     type: PRODUCTION
     name: Craft Anvils
-    production_time: 5s
+    production_time: 10s
     input:
-      iron blocks:
+      iron block:
```
Comment (Maxopoly) [inline configCivcraft.yml]: No spaces in identifierr, same for all the stuff below

Comment (Goldmattress) [inline configCivcraft.yml]: That's not an identifier, it's a display name, there are hundreds of them with spaces.

Comment (Maxopoly) [inline configCivcraft.yml]: Hmm last time I tried using spaces in yaml identifers it crashed, but I just ran this config through a validator and it seems fine, not sure. The yaml identifier is no longer the identifier used ingame, we now have something centralized for that. You can use whatever you want for the yaml structure, as long as the identifiers are unique.

Comment (Goldmattress) [inline configCivcraft.yml]: So the way it is is fine?

Comment (Maxopoly) [inline configCivcraft.yml]: Should be, but dont bother to make those names pretty or anything in the future

Comment (ProgrammerDan) [inline configCivcraft.yml]: You can always just quote delimit the identifiers, if there's a concern.

**Pull request merged**